drv/i2c-devices/src/mwocp68.rs

```diff
@@ -27,13 +27,68 @@ pub struct Mwocp68 {
     mode: Cell<Option<pmbus::VOutModeCommandData>>,
 }
 
+#[derive(Copy, Clone, PartialEq)]
+pub struct FirmwareRev(pub [u8; 4]);
+
+#[derive(Copy, Clone, PartialEq, Default)]
+pub struct SerialNumber(pub [u8; 12]);
+
+//
+// The boot loader command -- sent via BOOT_LOADER_CMD -- is unfortunately odd
+// in that its command code is overloaded with BOOT_LOADER_STATUS.  (That is,
+// a read to the command code is BOOT_LOADER_STATUS, a write is
+// BOOT_LOADER_CMD.)  This is behavior that the PMBus crate didn't necessarily
+// envision, so it can't necessarily help us out; we define the single-byte
+// payload codes here rather than declaratively in the PMBus crate.
+//
```

Comment:
Nit: triple `///` to actually generate a docstring, which is highlighted differently for those of us using syntax highlighting

(Also, leading and trailing empty lines are unidiomatic, but I know it will take some prying to remove them from your hands)

Reply:
It's definitely not intended to be a doc comment, just an implementation comment -- which hopefully gives me a stay of execution on the added vertical spacing for readability. ;)